src/explorer/CosmosDBTreeItem.ts

```diff
@@ -48,6 +49,54 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
         });
     }
 
+    public async createChildImpl(showCreatingTreeItem: (label: string) => void): Promise<AzureTreeItem<ISiteTreeRoot>> {
+        const connectionToAdd = <string>await vscode.commands.executeCommand('cosmosDB.api.getDatabase');
+        if (!connectionToAdd) {
+            throw new UserCancelledError();
+        }
+
+        const workspaceConfig = vscode.workspace.getConfiguration(constants.extensionPrefix);
+        const allConnections = workspaceConfig.get<IConnections[]>(constants.configurationSettings.connections, []);
+        let connectionsUnit = allConnections.find((x: IConnections) => x.webAppId === this.root.client.id);
+        if (!connectionsUnit) {
+            connectionsUnit = <IConnections>{};
+            allConnections.push(connectionsUnit);
+            connectionsUnit.webAppId = this.root.client.id;
+        }
+
+        // tslint:disable-next-line:strict-boolean-expressions
+        connectionsUnit.cosmosDB = connectionsUnit.cosmosDB || [];
+        if (!connectionsUnit.cosmosDB.find((x: string) => x === connectionToAdd)) {
+            connectionsUnit.cosmosDB.push(connectionToAdd);
+            workspaceConfig.update(constants.configurationSettings.connections, allConnections);
+            const createdDatabase = new CosmosDBDatabase(this, connectionToAdd);
+            showCreatingTreeItem(createdDatabase.label);
+
+            const appSettingsToUpdate = "MONGO_URL";
+            const connectionStringValue = (<string>await vscode.commands.executeCommand('cosmosDB.api.getConnectionString', connectionToAdd));
+            await updateWebAppSetting(connectionsUnit.webAppId, appSettingsToUpdate, connectionStringValue);
+
+            if (this.contextValue === 'AddCosmosDBConnection') {
+                // tslint:disable-next-line:no-non-null-assertion
+                await this.parent!.refresh();
+            } else {
+                await this.refresh();
+            }
+
+            const ok: vscode.MessageItem = { title: 'Ok' };
+            const showDatabase: vscode.MessageItem = { title: 'Show Database' };
+            // Don't wait
+            vscode.window.showInformationMessage(`Database "${createdDatabase.label}" connected to Web App "${this.root.client.fullName}". Created "${appSettingsToUpdate}" App Setting.`, ok, showDatabase).then(async (result: vscode.MessageItem | undefined) => {
+                if (result === showDatabase) {
+                    vscode.commands.executeCommand('appService.RevealConnection', createdDatabase);
+                }
+            });
+
+            return createdDatabase;
+        }
+        throw new Error("Impossible to have more than one CosmosDB connection for one web app!");
```

Comment:
Ok, I'll make a different PR for this, but in case If we continue to allow to have multiple connections should we:
1. just ignore if a user wants to add a connection
2. throw an error that for now only one connection possible to add
3. add this connection:
- MONGO_URL stays the same
- MONGO_URL has the value of the last added connection
And what should we do in case if a user deleted a connection that was the value of MONGO_URL, should connection stay the same or we need to delete it or replace with some other connection?

Reply:
Actually, it was a bug. I go there only if the database is already connected and exists in workspace settings. 

Anyway, I need to return/throw something in this func(it was void before refactoring), so I don't really can leave it without any changes.

So, what do you think is the best way to resolve this? Here are options within I choose:
1. Show notification that database connected successfully
2. Show notification(or error) that database already connected
3. Just do nothing (throw new UserCancelledError) 

For now, I just going to leave 3 variant, I think it's equivalent to 'no major behavior change'